### PR TITLE
driver.rkt: support network configuration

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

**Comment by blalor** [comment]
@nak3 Thank you for taking this on!

File: client/driver/rkt.go [hunk 1]
```diff
@@ -250,6 +263,63 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 		cmdArgs = append(cmdArgs, fmt.Sprintf("--dns-search=%s", domain))
 	}
 
+	// set network
+	network := ""
+	for _, n := range driverConfig.Net {
+		network += fmt.Sprintf("%s,", n)
+	}
+	// Trim last comma
+	network = strings.TrimRight(network, ",")
```
Comment (schmichael) [inline client/driver/rkt.go]: `network := strings.Join(driverConfig.Net, ",")` would be preferred.

Comment (nak3) [inline client/driver/rkt.go]: @schmichael Thank you. I updated it.

File: client/driver/rkt_test.go [hunk 2]
```diff
@@ -392,3 +392,49 @@ func TestRktTaskValidate(t *testing.T) {
...
+	task := &structs.Task{
+		Name: "etcd",
+		Config: map[string]interface{}{
+			"trust_prefix": "coreos.com/etcd",
+			"image":        "coreos.com/etcd:v2.0.4",
+			"command":      "/etcd",
+			"args":         []string{"--version"},
+			"port_map": []map[string]string{
+				map[string]string{
+					"main": "8080-tcp",
```
Comment (schmichael) [inline client/driver/rkt_test.go]: Don't you just mean 8080? This won't parse correctly.

**Comment by schmichael** [comment]
I rebased onto master and tried to fix your tests. If my PR #1862 looks ok we can just close this and have rkt port mappings in 0.5!

**Comment by nak3** [comment]
Thank you @schmichael 
Please close this PR and go ahead yours.

**Pull request closed without merging**